artifactory/utils/pip/pip.go

```diff
@@ -45,6 +51,35 @@ func (pc *PipCmd) GetErrWriter() io.WriteCloser {
 	return pc.ErrWriter
 }
 
+func GetPipDepTreeScriptPath() (string, error) {
```

Comment:
I suggest shortenning the function name to *GetDepTreeScriptPath* (*pip* is already the package name, so we don't need it as part of the function name).